share/github-backup-utils/ghe-ssh

```diff
@@ -61,7 +61,10 @@ if [ -z "$GHE_DISABLE_SSH_MUX" ]; then
   controlpath="$TMPDIR/.ghe-sshmux-$(echo -n "$user@$host:$port" | git hash-object --stdin | cut -c 1-8)"
   opts="-o ControlMaster=auto -o ControlPath=\"$controlpath\" -o ControlPersist=10m -o ServerAliveInterval=10 $opts"
   # Workaround for https://bugzilla.mindrot.org/show_bug.cgi?id=1988
-  [ -S $controlpath ] || ssh -f -p $port $opts -o BatchMode=yes "$host" -- /bin/true 1>/dev/null 2>&1 || true
+  if ! [ -S $controlpath ]; then
```

Comment:
Should we run the `cd` in a subshell so the working directory is not changed for the script itself? That side effect might trip up people in the future otherwise.

Reply:
Ah, good point.